src/popover/docs/demo.html

```diff
@@ -14,7 +14,7 @@
     </div>
     <button popover="{{dynamicPopover.content}}" popover-title="{{dynamicPopover.title}}" class="btn btn-default">Dynamic Popover</button>
 
-    <button popover-template="{{dynamicPopover.templateUrl}}" popover-title="{{dynamicPopover.title}}" class="btn btn-default">Popover With Template</button>
+    <button popover-template="dynamicPopover.templateUrl" popover-title="{{dynamicPopover.title}}" class="btn btn-default">Popover With Template</button>
```

Comment:
Just checked that change - it would now look under `dynamicPopover.content` for a template's `id` but there is none, so it would miss it